lib/util/esWrapper.js

```diff
@@ -116,7 +116,13 @@ const errorMessagesMapping = [
     regex: /^\[(.*)\]: version conflict, document already exists.*/,
     subcode: 'document_already_exists',
     getPlaceholders: matches => [matches[1]]
-  }
+  },
+  {
+    // Unknown key for a START_OBJECT in [term].
+    regex: /^Unknown key for a START_OBJECT in \[(.*)\].*/,
+    subcode: 'cannot_parse_query',
+    getPlaceholders: matches => [matches[1]]
+  },
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this trailing comma. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATrailingComma)